atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schemas.java

```diff
@@ -105,30 +104,19 @@ public static boolean isTableNameValid(String tableName) {
      * (e.g., it is not the responsibility of this method to perform schema
      * upgrades).
      */
-    public static void createTablesAndIndexes(Schema schema, KeyValueService kvs) {
+    public static void createTablesAndIndexes(Schema schema, DdlManager ddlManager) {
         schema.validate();
-        createTables(kvs, schema.getTableDefinitions());
-        createIndices(kvs, schema.getIndexDefinitions());
+        createTables(ddlManager, schema.getTableDefinitions());
+        createIndices(ddlManager, schema.getIndexDefinitions());
     }
 
-    public static void deleteTablesAndIndexes(Schema schema, KeyValueService kvs) {
-        schema.validate();
-        kvs.dropTables(getExistingTablesAlsoPresentInSchema(schema, kvs));
-    }
+    // To be removed
```

Comment:
That said, I think it makes sense to do a split here. So, my proposal would be:

1. deprecate this, provide `createUserManagedTable` or similar
2. wait 6 months, remove (maybe earlier if we see everyone has stopped using it)

Reply:
Alright, I'm on board with that plan, will adjust the PR.

Reply:
As a FLUP, though I won't track this now, it would be good to think through: this pattern of like barebones interface (DdlManager) and an ACTUALLY USEFUL factory class with static methods (Schemas) is a bit annoying.

Should we have either DdlManager or something else basically work at the level of either Schema or TableDefinition or something.